config/kernelci.conf

```diff
@@ -25,8 +25,10 @@ lab_config: shell
 [kcidb]
 
 [test_report]
-email_host: your.host
-email_port: your.port
+email_host: mail.collabora.com
+email_port: 587
```

Comment:
Please drop these changes.  In fact I think the whole section should be removed, but that's beyond the scope of this PR.

Reply:
Oh, it is committed by mistake. I added it for testing only.